### PR TITLE
Filter dashboard by site

### PR DESCRIPTION
**Pull request by twinsfan421**
## 💅🏼 What issue does this fix?
Addresses issue #2379  by adding a business filter to the Dashboard.  It is the same business filter used on the attendance page

## 🍂 Before You Submit

* [x] Did you write & run tests and linters?
* [x] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [ ] Migrations
* [ ] Dependencies

### CODE REVIEW

**Comment by twinsfan421** [comment]
@cjhaddad Let me know if you want me to change anything

File: .rubocop.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ Metrics/ClassLength:
   CountAsOne: ['array', 'heredoc', 'hash']
   Exclude: 
     - app/models/service_day.rb
+    - app/controllers/api/v1/users_controller.rb
```
Comment (fabianraf) [inline .rubocop.yml:26]: Why is this file being excluded?

File: client/src/Dashboard/Dashboard.js [hunk 2]
```diff
@@ -256,12 +258,19 @@ export function Dashboard() {
...
   }
 
-  const getDashboardData = async (filterDate = undefined) => {
+  const getDashboardData = async ({
+    businessIds = [],
+    filterDate = undefined
+  } = {}) => {
     dispatch(setLoading(true))
-    const baseUrl = '/api/v1/case_list_for_dashboard'
+    let baseUrl = `/api/v1/case_list_for_dashboard`
```
Comment (fabianraf) [inline client/src/Dashboard/Dashboard.js:266]: Why was this change needed? I don't see baseUrl changing anywhere. Maybe I'm missing something?

Comment (twinsfan421) [reply client/src/Dashboard/Dashboard.js]: This change wasn't needed, I had planed to change that more and then didn't need to.  It was an error on my part

File: .rubocop.yml (hunk 1 above)
Comment (twinsfan421) [reply .rubocop.yml]: The Rubocop error is for Class lengths that are over 100 lines and the changes to users_controller made it over 100 lines(101 lines).  I just pushed up a commit to make the filter_date method a ternary operator to get it under the 100 lines.  Another option is I could move the filter date and the dashboard_query methods into a service_object if preferred.

File: client/src/Dashboard/Dashboard.js (hunk 2 above)
Comment (twinsfan421) [reply client/src/Dashboard/Dashboard.js]: @fabianraf Is there any more changes you want me to make to get this PR approved?

Comment (fabianraf) [reply client/src/Dashboard/Dashboard.js]: Sorry, I thought I had approved already. Approving it now!

**Review by fabianraf — APPROVED** [review]
